pkgs/applications/misc/parsec/bin.nix

```diff
@@ -0,0 +1,81 @@
+{ stdenvNoCC, stdenv
+, lib
+, dpkg, autoPatchelfHook, makeWrapper
+, fetchurl
+, alsa-lib, openssl, udev
+, libglvnd
+, libX11, libXcursor, libXi, libXrandr
+, libpulseaudio
+, libva
+, ffmpeg
+}:
+
+stdenvNoCC.mkDerivation {
+  pname = "parsec-bin";
+  version = "150_28";
+
+  src = fetchurl {
+    url = "https://web.archive.org/web/20220622215230id_/https://builds.parsecgaming.com/package/parsec-linux.deb";
+    sha256 = "1hfdzjd8qiksv336m4s4ban004vhv00cv2j461gc6zrp37s0fwhc";
+  };
+
+  unpackPhase = ''
+    runHook preUnpack
+
+    dpkg-deb -x $src .
+
+    runHook postUnpack
+  '';
+
+  nativeBuildInputs = [ dpkg autoPatchelfHook makeWrapper ];
+
+  buildInputs = [
+    stdenv.cc.cc # libstdc++
+    libglvnd
+    libX11
+  ];
+
+  runtimeDependenciesPath = lib.makeLibraryPath [
+    stdenv.cc.cc
+    libglvnd
+    openssl
+    udev
+    alsa-lib
+    libpulseaudio
+    libva
+    ffmpeg
+    libX11
+    libXcursor
+    libXi
+    libXrandr
+  ];
+
+  prepareParsec = ''
+    if [[ ! -e ~/.parsec/appdata.json ]]; then
+      mkdir -p ~/.parsec
+      cp -r --no-preserve=mode,ownership,timestamps ${placeholder "out"}/share/parsec/skel/* ~/.parsec/
+    fi
```

Comment:
Can you change `~` to `$HOME`? Also, please properly quote everything, since we different from `/nix/store` we can't assume that there will be no spaces between the path.